ccd-definition/ComplexTypes/CareSupervision/AdditionalApplicationsBundle.json

```diff
@@ -0,0 +1,49 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "AdditionalApplicationsBundle",
+    "ListElementCode": "c2DocumentBundle",
+    "FieldType": "C2DocumentBundle",
+    "ElementLabel": "C2 application",
+    "SecurityClassification": "Public",
+    "Searchable": "N"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "AdditionalApplicationsBundle",
+    "ListElementCode": "otherApplicationsBundle",
+    "FieldType": "OtherApplicationsBundle",
+    "ElementLabel": "Other applications",
+    "SecurityClassification": "Public",
+    "Searchable": "N"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "AdditionalApplicationsBundle",
+    "ListElementCode": "pbaPayment",
+    "FieldType": "PBAPayment",
+    "ElementLabel": "PBA Payment",
+    "SecurityClassification": "Public",
+    "Searchable": "N"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "AdditionalApplicationsBundle",
+    "ListElementCode": "uploadedDateTime",
+    "FieldType": "Text",
+    "ElementLabel": "Date and time of upload",
+    "SecurityClassification": "Public",
+    "FieldShowCondition": "c2DocumentBundle=\"Do Not Show\"",
```

Comment:
there is convention that it is all capital 'DO NOT SHOW'

Reply:
updtaed